### DISCUSSION TITLE
Upgraded to latest theme, and ran into two issues (Summary's short, and missing css files)

### DISCUSSION THREAD

**Discussion by zwoop** [razonyang/hugo-theme-bootstrap#1007] [Q&A]
Hi,

Hopefully it's just me doing something dumb, but I upgraded to the latest version, and ran into two problems:

**Missing CSS file?**

```
Error: error building site: TOCSS: failed to transform "main/scss/main.css" (text/css): "themes/hugo-theme-bootstrap/assets/main/scss/_bootstrap.scss:7:1": File to import not found or unreadable: bootstrap/scss/variables-dark.
```

if I comment out these two lines, things at least builds:

```
// @import 'bootstrap/scss/variables-dark';
// @import 'bootstrap/scss/maps';
```

**Summary is now really short (is it because of the above??)**

The summaries are now much shorter than before, and I can't figure out to make them longer. They will be a few words long, and never more than one single line. This one I haven't been able to figure out, but I'm assuming it's related to the upgrade, and something I'm doing wrong (going back to my older version of the theme fixes it).

Help? :)

**Comment by razonyang** [comment]
Hi @zwoop, sorry for the inconvenience.

> Missing CSS file?

I guess you're using a old Bootstrap version that doesn't have those SASS files, please perform update and try again, otherwise the dark mode won't work.

> `npm update` should be performed each time after upgrading the theme.

```sh
npm update
```

> Summary is now really short (is it because of the above??)

The newer version uses the `description` (on front matter) as the summary if not empty, please let me know if you want the old behavior,  will provide an option.

**Comment by zwoop** [comment]
Ah, thank you so much! Yeh, the npm update fixed it, I don't know why I can't remember to do that, I know I've run into that before too ... Thanks again!

As for the summary, description makes sense. I think it'd be really nice to have an option to allow for the old summary behavior, but I can work with the current behavior (just have to go back and make the description a little longer and more detailed.

**Comment by razonyang** [comment]
Done, [v1.5.0](https://github.com/razonyang/hugo-theme-bootstrap/releases/tag/v1.5.0) brings the `post.excerpt` param back, to use old behavior of summaries/excerpts, you'll need to specify it as empty string.

```
// params.toml
[post]
excerpt = ""
```

**Reply by zwoop** [reply]
sweet, thanks! Works great!
